.pylintrc

```diff
@@ -7,7 +7,13 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,
+       apply_adaround.py,
```

Comment:
Why are we skipping pylint for these source files?

Reply:
This is because they are pseudo-code, not real executable Python code at the moment.
[For example:](https://github.com/quic/aimet/pull/3471/files#diff-62393786a3cf6e481ff969ecde21cd8487f8ecdc51ec373736186baf37c237f2R40)
```py
prepared_model = prepare_model(<model>)
```